esp/src/src-react/components/SetLogicalFilePartAttr.tsx

```diff
@@ -0,0 +1,95 @@
+import * as React from "react";
+import { DefaultButton, DetailsList, DetailsListLayoutMode, IColumn } from "@fluentui/react";
+import { SizeMe } from "react-sizeme";
+import { csvParse } from "d3-dsv";
+import { DaliService } from "@hpcc-js/comms";
+import { scopedLogger } from "@hpcc-js/util";
+import { TableGroup } from "./forms/Groups";
+import { useConfirm } from "../hooks/confirm";
+import nlsHPCC from "src/nlsHPCC";
+import { HolyGrail } from "../layouts/HolyGrail";
+
```

Comment:
Whenever the checks run, I believe the ECL Watch build steps in GH actions will fail because of errors about "...more than 1 blank line not allowed...". But you should be able to take care of them all by running `npm run lint-fix`